t/apicast-async-reporting.t

```diff
@@ -1,202 +1,181 @@
 use lib 't';
-use Test::APIcast 'no_plan';
+use Test::APIcast::Blackbox 'no_plan';
 
-env_to_nginx('APICAST_REPORTING_THREADS=1');
+$ENV{APICAST_ACCESS_LOG_FILE} = "$Test::Nginx::Util::ErrLogFile";
+$ENV{TEST_NGINX_HTML_DIR} ||= "$Test::Nginx::Util::ServRoot/html";
+$ENV{APICAST_REPORTING_THREADS} = 1;
 
+check_accum_error_log();
+repeat_each(1);
 run_tests();
 
 __DATA__
 
 === TEST 1: api backend gets the request
 It asks backend and then forwards the request to the api.
---- http_config
-include $TEST_NGINX_UPSTREAM_CONFIG;
-lua_shared_dict api_keys 1m;
-lua_package_path "$TEST_NGINX_LUA_PATH";
-init_by_lua_block {
-  ngx.shared.api_keys:set('42:value:usage%5Bhits%5D=2', 200)
```

Comment:
the workaround that I'm using to simulate this is to actually fire 2 pipelined requests, so that the key is populated automatically by APIcast on the first and the second requests is reported asynchronously.